code/modules/mob/living/simple_animal/hostile/mining_mobs/elites/goliath_broodmother.dm

```diff
@@ -232,15 +232,34 @@
 // Broodmother's loot: Broodmother Tongue
 /obj/item/crusher_trophy/broodmother_tongue
 	name = "broodmother tongue"
-	desc = "The tongue of a broodmother.  If attached a certain way, makes for a suitable crusher trophy."
+	desc = "The tongue of a broodmother.  If attached a certain way, makes for a suitable crusher trophy.  It also feels very spongey, I wonder what would happen if you squeezed it?..."
 	icon = 'icons/obj/lavaland/elite_trophies.dmi'
 	icon_state = "broodmother_tongue"
 	denied_type = /obj/item/crusher_trophy/broodmother_tongue
 	bonus_value = 10
+	var/use_time
```

Comment:
[add dmdoc to meet documentation requirements](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md)